### PR TITLE
Sprint 2 front end

### LINKED ISSUE #44: Redesign Header/Footer Components Based on redesign
Make our header and footer look like #22

### LINKED ISSUE #49: Redesign Footer

### LINKED ISSUE #51: Implement Sassy CSS (SCSS) and SCSS Linter

### LINKED ISSUE #60: Remove Material UI
No longer using Material UI.

### LINKED ISSUE #62: Validate naming conventions
SPRINT-2 branch has class names still written as camelCase when they should be kebab-case. Styling selectors aren't hitting the desired classes.

eg: `footerText` -> `footer-text`

### LINKED ISSUE #71: Header font weight should be 400

### PR DESCRIPTION
**Pull request by rainrivas**
It is time to put Sprint 2 into development for all to see. I don't think I should be the one opening this, but meh. Go to the heroku test page to see it in action if you don't want to build locally.

## Issues Worked On
- Fixes #71 
- Fixes #62 
- Fixes #60 
- Fixes #51 
- Fixes #49 
- Fixes #44

### CODE REVIEW

**Review by jwu910 — DISMISSED** [review]
:fire:

File: src/components/Footer.js [hunk 1]
```diff
@@ -3,30 +3,28 @@ import links from '../constants/links';
 
 const Footer = (props) => {
   return (
```
Comment (phuchle) [inline src/components/Footer.js]: We can eliminate this return and use the implicit arrow return like

```
const Footer = (props) => (
  <Component />
)
```

File: src/styles/constants/shared.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Theme colors
```
Comment (phuchle) [inline src/styles/constants/shared.scss:1]: Can we rename these to include the general color? For example , $primary1-dark-red-color.

**Review by phuchle — CHANGES_REQUESTED** [review]
Overall it's good.  I'm requesting some minor changes.

File: src/styles/constants/shared.scss (hunk 2 above)
Comment (jwu910) [reply src/styles/constants/shared.scss:1]: Can we forego this change for this branch until we have a clearer idea of how we want to structure colors, variable names, and whether we will be using `CSS IN JS` ?

Comment (phuchle) [reply src/styles/constants/shared.scss:1]: Yup, I think it's fine to defer until we have a better idea of how we want to do things.

**Review by phuchle — APPROVED** [review]

**Comment by jwu910** [comment]
@phuchle [1e7f419 ](https://github.com/5-gwoap/adopt-a-family/pull/70/commits/1e7f4198073079f3330cac5b6029ce64ba532777) should be the component returns you wanted to see.

**Review by jwu910 — APPROVED** [review]
🌶

**Review by psyduc — APPROVED** [review]

**Review by AlChiu — APPROVED** [review]

**Comment by jwu910** [comment]
Merged, thanks!

**Pull request merged**